src/me/drton/flightplot/export/KmlExportFormat.java

```diff
@@ -11,13 +11,11 @@ public String getFormatName() {
 
     @Override
     public String getFileExtension() {
-        return ".kml";
```

Comment:
@kd0aij This doesn't work for me, and produces files without the dot before the extension.